systemjs/Plugin.php

```diff
@@ -27,24 +27,51 @@ public function appendConfiguration(ArrayNodeDefinition $rootNode)
                 ->end()
                 ->validate()
                     ->always(function($v) {
-                        $bin = 'systemjs-bundle';
-                        if (empty($v['bundle_js']) || !is_file($v['bundle_js'])) {
-                            if (is_file('node_modules/.bin/' . $bin)) {
-                                $v['bundle_js'] = 'node_modules/.bin/' . $bin;
-                            } elseif (false !== $paths = getenv('PATH')) {
-                                foreach (explode(PATH_SEPARATOR, $paths) as $path) {
-                                    if (is_file($path . DIRECTORY_SEPARATOR . $bin)) {
-                                        $v['bundle_js'] = realpath($path . DIRECTORY_SEPARATOR . $bin);
-                                        break;
-                                    }
-                                }
+                        $candidates = [
+                            // is it... manually specified
+                            isset($v['bundle_js']) ? $v['bundle_js'] : null,
+
+                            // is it... installed in the project (should be the case most of the times)
+                            'node_modules/.bin/systemjs-bundle',
+
+                            // is it... installed globally (should be for very old projects)
+                            '/usr/local/bin/systemjs-bundle',
+
+                            // it it... installed somewhere else entirely
+                            $this->getBundleJsFromPath(),
+                        ];
+
+                        $bundleJs = null;
+                        foreach ($candidates as $candidate) {
+                            if (!empty($candidate) && is_file($candidate)) {
+                                $bundleJs = $candidate;
+                                break;
                             }
                         }
 
+                        if ($bundleJs) {
+                            $v['bundle_js'] = $bundleJs;
+                        } else {
+                            throw new \Exception(sprintf('Unable to locate systemjs-bundle.  Looked in %s', json_encode($candidates)));
```

Comment:
Het zijn strings, kan ook gewoon:
```suggestion
                            throw new \Exception(sprintf('Unable to locate systemjs-bundle.  Looked in %s', implode(', ', $candidates)));
```
Eventueel een array_filter()-tje er overheen om de null waarden te filteren